runtime/src/lib.rs

```diff
@@ -313,14 +307,15 @@ impl pallet_contracts::Config for Runtime {
 	type WeightPrice = pallet_transaction_payment::Pallet<Self>;
 	type WeightInfo = pallet_contracts::weights::SubstrateWeight<Self>;
 	type ChainExtension = LocalChainExtension;
-	type DeletionQueueDepth = DeletionQueueDepth;
+	type DeletionQueueDepth = ConstU32<128>;
```

Comment:
Parameters matches the recent Shiden parameters
https://github.com/AstarNetwork/Astar/pull/708/files#diff-80518c95a1f023b8f716bf975c7a6ff78d4c6cc85e4363fa108e95d3b3b3c1c3R612